contrib/blueflood-docker/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+blueflood:
+  build: .
+  links:
+    - cassandra
+    - elasticsearch
+
+cassandra:
+  image: cassandra:2.1
+
+cassandran:
```

Comment:
No pun intended :grinning:  "n" essentially means you can scale this to n number of containers, although this is useless here because node discoveryType is set to none in Bluefloods' configuration. One has to change it to TOKEN_AWARE or or RING_DESCRIBE to actually leverage this feature of Docker compose.
BTW do let me know about the possible pun; I am such an ignorant soul :stuck_out_tongue: